lib/parser/provenance.cc

```diff
@@ -228,18 +228,18 @@ void AllSources::EmitMessage(std::ostream &o,
           [&](const Inclusion &inc) {
             o << inc.source.path();
             std::size_t offset{origin.covers.MemberOffset(range->start())};
-            std::pair<int, int> pos{inc.source.FindOffsetLineAndColumn(offset)};
-            o << ':' << pos.first << ':' << pos.second;
+            SourcePosition pos{inc.source.FindOffsetLineAndColumn(offset)};
```

Comment:
Should the `SourceFile` in the `SourcePosition` be emitted, or checked, or both?

Reply:
What about `SourceFile` do you think should be emitted or checked? Currently the `SourceFile`'s path is already emitted through `o << inc.source.path();`.

Reply:
That's fine, then.